gtsam.h

```diff
@@ -874,7 +874,7 @@ class Cal3_S2 {
   double py() const;
   gtsam::Point2 principalPoint() const;
   Vector vector() const;
-  Matrix matrix() const;
+  Matrix K() const;
```

Comment:
Deprecated but can’t break existing python code in GTSAM 4. Feel free to add K, but leave matrix?

Reply:
All done.